src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -107,7 +107,11 @@ protected function deleteExisting(CanResetPasswordContract $user)
      */
     protected function getPayload($email, $token)
     {
-        return ['email' => $email, 'token' => $this->hasher->make($token), 'created_at' => new Carbon];
+        return [
+            'email' => $email,
+            'token' => $this->hasher->make($token),
+            'created_at' => app(Factory::class)->now(),
```

Comment:
This is just used to store "now" as `created_at` for `reset_passwords`, `Carbon::now()` should be more than enough.